atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -183,10 +185,10 @@ long grabLocksAndRun(SweepLocks locks) throws InterruptedException {
                 if (locks.haveLocks()) {
                     sweptSuccessfully = runOnce();
                 } else {
-                    log.debug("Skipping sweep because sweep is running elsewhere.");
+                    log.warn("Skipping sweep because sweep is running elsewhere.");
                 }
             } else {
-                log.debug("Skipping sweep because it is currently disabled.");
+                log.info("Skipping sweep because it is currently disabled.");
```

Comment:
This will keep spamming the logs if the config flag is disabled. Will leave it at debug.